src/qw/cli.py

```diff
@@ -145,31 +175,46 @@ def check(
     issue: Annotated[
         Optional[int],
         typer.Option(
-            help="Issue number to check",
+            help="Issue number to check (default is all of them)",
+            show_default=False,
```

Comment:
interesting, I'm assuming the default is something a bit strange or `None`?

Reply:
Yeah, it's just None. Which is the wrong thing from a speaking English point of view, but I think it's probably unsurprising to Python programmers.